src/components/ui/post/detail/RoomTourModal.tsx

```
@@ -55,6 +63,28 @@ export default function RoomTourModal() {
+		const date = new Date(selectedDate);
+		let hour24 = hour % 12;
+		if (period === 'PM') hour24 += 12;
+		date.setHours(hour24, minute, 0, 0);
+
+		try {
+			await RoomTourService.createRoomTour(client, {
+				boardingRoomId,
+				time: date.toISOString(),
+			});
+			onSuccess?.();
+		} catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)